### ISSUE TITLE
Bug in in the elbo function

### ISSUE THREAD

**Issue by caghankoksal** [barisbatuhan/SSuperGAN#25]
Calling parameters for the reconstruction_loss distributional is buggy.

`def elbo(z,
         x,
         mu_z,
         mu_x,
         logstd_z,
         logstd_x=None,
         kl_loss_weight=1,
         recon_loss_weight=1,
         l1_recon=False):
    kl_loss_value = kl_loss(z, mu_z, torch.exp(logstd_z))
    if not l1_recon:
        reconstruction_loss = -recon_loss_weight * reconstruction_loss_distributional(x,mu_x, logstd_x)
    else:
        reconstruction_loss = recon_loss_weight * l1_loss(mu_x, x)
    loss = reconstruction_loss + kl_loss_weight * kl_loss_value

    return OrderedDict(loss=loss,
                       reconstruction_loss=reconstruction_loss,
                       kl_loss=kl_loss_value)`
